app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallViewModel.kt

```diff
@@ -140,14 +138,6 @@ class OngoingCallViewModel @AssistedInject constructor(
         }
     }
 
-    private fun mapQualityStream(uiParticipant: UICallParticipant): CallQuality {
```

Comment:
question: is removing call quality somehow related to deleting group conversation locally? What is the reason for this change? 🤔

Reply:
Its not, but it is required to get latest kalium develop

Reply:
reverted, on the latest develop the quality is thjere, probably the changes were not having that change at the moment when I created this pr